src/main/java/org/broadinstitute/hellbender/tools/exome/eval/EvaluationSampleSummaryReader.java

```diff
@@ -0,0 +1,28 @@
+package org.broadinstitute.hellbender.tools.exome.eval;
+
+import org.broadinstitute.hellbender.utils.tsv.DataLine;
+import org.broadinstitute.hellbender.utils.tsv.TableReader;
+
+import java.io.File;
+import java.io.IOException;
+import java.io.Reader;
+
+/**
+ * Table reader for {@link EvaluationSampleSummaryRecord} instances.
+ */
+public final class EvaluationSampleSummaryReader extends TableReader<EvaluationSampleSummaryRecord> {
+
+    public EvaluationSampleSummaryReader(final File file) throws IOException {
+        super(file);
+    }
+
+    @Override
+    protected EvaluationSampleSummaryRecord createRecord(final DataLine dataLine) {
+        final String sample = dataLine.get(EvaluationSampleSummaryWriter.SAMPLE_NAME_COLUMN_NAME);
+        final EvaluationSampleSummaryRecord result = new EvaluationSampleSummaryRecord(sample);
+        for (final EvaluationClass clazz : EvaluationClass.values()) {
```

Comment:
class would compile as is a keyword. I can change to evalClass or evaluationClass if you prefer.